### PR TITLE
Update Tier-1 Data documentation

### PR DESCRIPTION
**Pull request by jefscheepers**
The Tier-1 Data documentation was out of date on some points:

- Login procedure for Python client
- Overview of clusters with iCommands installation
- Group administation workflow
- Documentation of the search function
- ...

This PR fixes most of these issues.

### CODE REVIEW

File: source/data/tier1data/clients/mango_portal.rst [hunk 1]
```diff
@@ -202,4 +202,33 @@ This feature is a proof of concept, and this information can currently not be ad
 Searching
 =========
 
-**Work in progress**
+The ManGO portal contains both a quick search bar and an advanced search form. 
+
+The search bar is located on top of the webpage.
```
Comment (ingridbr) [inline source/data/tier1data/clients/mango_portal.rst:207]: The quick search bar is located on top of the webpage

Comment (ingridbr) [inline source/data/tier1data/clients/mango_portal.rst:207]:
```suggestion
The quick search bar is located on top of the webpage.   
```

File: source/data/tier1data/clients/mango_portal.rst [hunk 2]
```diff
@@ -202,4 +202,33 @@ This feature is a proof of concept, and this information can currently not be ad
...
+The advanced search tab is located on the left side of the screen.
+This brings you to a form where you can search for data objects and collection based on several criteria.
+You can also search based on any metadata fields.
+
+If you want to search based on schema metadata, you will need to use the following convention:
+
+- Attribute name: mgs.<schema_id>.<field_id>
+- Attribute value: <value>
+
+For example, if you want to search based on the field 'instrument' from the schema 'chemistry', you will need to search for attribute name 'mgs.chemistry.experiment'.
```
Comment (ingridbr) [inline source/data/tier1data/clients/mango_portal.rst:230]: I think 'mgs.chemistry.experiment'. should be 'mgs.chemistry.instrument'.

File: source/data/tier1data/features/group_administration.rst [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+- Support
+
+The **responsible**, as the name suggest, is the primary responsible for the project. They are the primary point of contact for the support staff in administrative issues.
+The **manager(s)** has the power to add and remove project members and to modify subgroups (see later).
+
+The support role is meant for support staff from the VSC. 
+They might temporarily join projects when they are actively involved, or for debugging purposes.
+
+To manage project members, reponsibles/managers need to go to the web page of their project, 
+by going to https://mango.vscentrum.be/, clicking on their zone to expand the list of projects they are, and clicking on their project name.
```
Comment (ingridbr) [inline source/data/tier1data/features/group_administration.rst:38]:
```suggestion
by going to https://mango.vscentrum.be/, clicking on their zone to expand the list of projects they are member of, and clicking on their project name.

```

File: source/data/tier1data/features/group_administration.rst [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+After making one or multiple of these changes, a button 'apply changes in iRODS' will appear.
+Click on this button to finalize your changes. 
+
+
+
+******************
+Managing subgroups
+******************
+
+Subgroups are used to gives permissions to a specific group of members from your project.
```
Comment (ingridbr) [inline source/data/tier1data/features/group_administration.rst:61]:
```suggestion
Subgroups are used to give permissions to a specific group of members from your project.

```

**Review by ingridbr — CHANGES_REQUESTED** [review]

File: source/data/tier1data/features/group_administration.rst [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+- Responsible
+- Manager
+- Member
+- Support
+
+The **responsible**, as the name suggest, is the primary responsible for the project. They are the primary point of contact for the support staff in administrative issues.
+The **manager(s)** has the power to add and remove project members and to modify subgroups (see later).
+
+The support role is meant for support staff from the VSC. 
+They might temporarily join projects when they are actively involved, or for debugging purposes.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:35]: This is better suited to be presented with a *definition* formatting.

```suggestion
Responsible
    Primary responsible for the project. They are the primary point of contact for the support staff in administrative issues.

Manager
    Role with the power to add and remove project members and to modify subgroups (see later).

Member
    Regular role without any special permissions.
    
Support
     Support staff from the VSC who might temporarily join projects when they are actively involved, or for debugging purposes.

```

File: source/data/tier1data/features/group_administration.rst (hunk 3 above)
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:38]:
```suggestion
by going to the `ManGO portal <https://mango.vscentrum.be/>`_, clicking on their zone to expand the list of projects they are, and clicking on their project name.
```

File: source/data/tier1data/features/group_administration.rst [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+To manage project members, reponsibles/managers need to go to the web page of their project, 
+by going to https://mango.vscentrum.be/, clicking on their zone to expand the list of projects they are, and clicking on their project name.
+Alternatively, they can go straight to *https://mango.vscentrum.be/data-platform/project/<project_name>*.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:39]:
```suggestion
Alternatively, they can go straight to ``https://mango.vscentrum.be/data-platform/project/<project_name>``.

```

File: source/data/tier1data/features/group_administration.rst [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+On this page, you get an overview of all members in the project:
+
+.. figure:: ../images/group_administration/mango_portal_manage_members.png
+  :alt: An overview of all members.
+
+- To add a new member to the project, click on 'add member' and search them by their name or account number. Next, click on 'Add'.
+
+- To change the role of a member, click on the blue pencil icon next to their name. Select their new role on the dropdown, and click on 'Update'.
+
+- To remove a member from the project, click on the red trashcan icon next to their name. In the menu that pops up, click on 'Delete'.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:50]: I suggest to put the names of the clickables in italics or bold.

File: source/data/tier1data/features/group_administration.rst [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+- The main group of your project  
+- The 'managers' group  
+- ...
+
+Other groups, which you can create yourself, are called 'custom groups'.
+For example, you could create a custom group for staff and one for master students. 
+To do so, click on 'Add group' and give a name.
+Custom groups will always get your project name as prefix. For example, if your project is called 'demo-project' and you enter 'master_students', the custom group will be called 'demo-project_master_students'.
+
+When you create a new group, or where you click on one, you will get an overview of the group members on the left side.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:77]:
```suggestion
Whenever you create a new group or access an existing one, you will get an overview of the group members on the left side.

```

File: source/data/tier1data/features/group_administration.rst [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+
+
+.. figure:: ../images/group_administration/mango_portal_group_administration.png
+  :alt: Editing the members of a custom group
+
+To add a member to the group, select their checkbox and click on 'Add selected users'.
+
+Likewise, to remove users from the group, select their checkbox and click on 'Remove selected users'.
+
+**Note:** You may need to log out and login again before you see changes you made in groups reflected when editing permissions.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:88]: Use proper note formatting:

```suggestion
.. note::

   You may need to log out and login again before you see changes you made in groups reflected when editing permissions. 

```

File: source/data/tier1data/features/group_administration.rst [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+To add a member to the group, select their checkbox and click on 'Add selected users'.
+
+Likewise, to remove users from the group, select their checkbox and click on 'Remove selected users'.
+
+**Note:** You may need to log out and login again before you see changes you made in groups reflected when editing permissions. 
+
+
+
+Help! I want to collaborate with an external researcher
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:92]: Better to not incite users to scream at us :slightly_smiling_face: 
```suggestion
How to collaborate with an external researcher

```

File: source/data/tier1data/features/group_administration.rst [hunk 11]
```diff
@@ -0,0 +1,125 @@
...
+-------------------------------------------------------
+
+As demonstrated earlier, you can add collaborators to your group if they have a VSC account
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:95]:
```suggestion
As demonstrated earlier, you can add collaborators to your group if they have a VSC account.

```

File: source/data/tier1data/features/group_administration.rst [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+If you want to work together with someone who doesn't have a VSC account, there are two options:
+
+1) Applying for a VSC account
+
+You can let your collaborators apply for a VSC account.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:100]: This is not always possible. I would rephrase this to something like:

> If your collaborators are eligible for a VSC account, we recommend that they apply for one to access Tier-1 data.

File: source/data/tier1data/features/group_administration.rst [hunk 13]
```diff
@@ -0,0 +1,125 @@
...
+
+- an institutional account from any institute that is connected to Globus
+- an ORCID ID 
+- a Gmail account
+
+Via a so-called guest collection, you can share data from different storage systems -including Tier-1 Data- with externals.
+You can give users or groups of users either read or write access on your data.
+However, there are some caveats:
+
+- Users will only be able to download or upload data via the Globus interface, and not via any of the other Tier-1 Data clients.
```
Comment (lexming) [inline source/data/tier1data/features/group_administration.rst:119]: `Tier-1 Data Clients` should be a link to the section in VSC docs where this clients are presented.

File: source/data/tier1data/clients/mango_portal.rst [hunk 14]
```diff
@@ -202,4 +202,33 @@ This feature is a proof of concept, and this information can currently not be ad
 Searching
 =========
 
-**Work in progress**
+The ManGO portal contains both a quick search bar and an advanced search form. 
+
+The search bar is located on top of the webpage.   
+You can find data objects and collections with it, by searching based on one of the following criteria:
```
[the remark this answers is not in the document]
Comment (jefscheepers) [reply source/data/tier1data/clients/mango_portal.rst:208]: It seems you are correct. If you give multiple terms, collections/data objects matching any of them will be picked up

File: source/data/tier1data/clients/python_client.rst [hunk 15]
```diff
@@ -48,12 +48,8 @@ There are three ways to authenticate with the Python-iRODSclient:
    up. You might need to put the file in a folder that doesn't require administrator rights.  
 
 3. Linux users can first authenticate with :ref:`icommands`. 
-   Then, they create a second configuration file as follows:
+   The user is also authenticated for the Python-iRODSClient.
```
[the remark this answers is not in the document]
Comment (jefscheepers) [reply source/data/tier1data/clients/python_client.rst:51]: Yes, that is correct.
You do need to put a small code snippet in your script to make use of this session (https://github.com/irods/python-irodsclient#establishing-a-secure-connection), but you don't need to authenticate separately for the Python-irodsclient.

File: source/data/tier1data/clients/mango_portal.rst (hunk 1 above)
Comment (jefscheepers) [reply source/data/tier1data/clients/mango_portal.rst:207]: Done

File: source/data/tier1data/clients/mango_portal.rst (hunk 2 above)
Comment (jefscheepers) [reply source/data/tier1data/clients/mango_portal.rst:230]: Silly one, thanks. Done

File: source/data/tier1data/features/group_administration.rst (hunk 3 above)
Comment (jefscheepers) [reply source/data/tier1data/features/group_administration.rst:38]: done

File: source/data/tier1data/features/group_administration.rst (hunk 4 above)
Comment (jefscheepers) [reply source/data/tier1data/features/group_administration.rst:61]: done

File: source/data/tier1data/features/group_administration.rst (hunk 12 above)
Comment (jefscheepers) [reply source/data/tier1data/features/group_administration.rst:100]: Thanks, I didn't realize that. I will phrase it as 'they can' instead of 'we recommend' (because we recommend either a VSC account or Globus depending on the context).

**Review by lexming — APPROVED** [review]
LGTM

**Review by ingridbr — APPROVED** [review]
LGTM
